app/views/article/_form.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="text-center">
+	<main class="form-signin">
+		<%=form_with model:article do |f| %>
+			<img class="mb-4" src="/docs/5.1/assets/brand/bootstrap-logo.svg" alt="" width="72" height="57">
+			<h1 class="h3 mb-3 fw-normal">Please sign in</h1>
+			<div class="form-floating">
+				<%=f.text_field :title, class:'form-control', placeholder: 'Title of Article' value:((article.title.nil?)?'':article.title)%>
```

Comment:
(article.title.nil? ? '' : article.title)

and maintain consistency while spacing as after class there is no space but after placeholder you added space